### PR TITLE
Set torch.backends.cudnn.enabled to false when testing accuracy

### PR DESCRIPTION
**Pull request by desertfire**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #94363

Summary: It looks like setting torch.backends.cudnn.deterministic to
True is not enough for eliminating non-determinism when testing
benchmarks with --accuracy, so let's turn off cudnn completely.
With this change, mobilenet_v3_large does not show random failure on my
local environment. Also take this chance to clean up CI skip lists.

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx

### CODE REVIEW

File: benchmarks/dynamo/common.py [hunk 1]
```diff
@@ -112,13 +112,8 @@ class CI(NamedTuple):
...
     "cait_m36_384",  # fp64_OOM
     "convit_base",  # fp64_OOM
-    "fbnetv3_b",  # Accuracy (blocks.2.2.bn1.weight.grad)
-    "levit_128",  # Accuracy (patch_embed.0.c.weight.grad)
-    "sebotnet33ts_256",  # Accuracy (stem.conv1.conv.weight.grad)
-    "xcit_large_24_p8_224",  # fp64_OOM,
-    "gernet_l",  # accuracy https://github.com/pytorch/pytorch/issues/93847
-    "gluon_xception65",  # accuracy https://github.com/pytorch/pytorch/issues/93847
-    "tinynet_a",  # accuracy https://github.com/pytorch/pytorch/issues/93847
+    "sebotnet33ts_256",  # Accuracy (stages.1.1.attn.fc1.bias.grad), how come inductor passes?
```
Comment (ezyang) [inline benchmarks/dynamo/common.py:115]: ikr, so perplexing

**Review by ezyang — APPROVED** [review]
OK, but cc @kurtamohler, it really should be deterministic...

**Comment by desertfire** [comment]
Sigh, there is still inconsistency between local-run results and CI results.

**Pull request closed without merging**